### PR TITLE
Updating Storebuilder shipping wizard UI

### PR DESCRIPTION
**Pull request by aykanburcak**
### Jira Ticket
[SITE-91](https://moderntribe.atlassian.net/browse/SITE-91)

### Description
This PR updates the Storebuilder shipping wizard UI by the wme-ui V2 components.

### Screenshot
Screen Capture on 2023-01-12 at 21-37-52

[SITE-91]: https://moderntribe.atlassian.net/browse/SITE-91?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/storebuilder/src/wizards/WizardHeader.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { WizardHeader as WmeWizardHeader, Logo, ExitButton } from '@moderntribe/wme-ui';
 import { __ } from '@wordpress/i18n';
 import { useWizard } from '@store/hooks';
-import { StoreBuilderLogo } from '@store/logos';
+import { NexcessLogo } from '@store/logos';
```
Comment (chumpgrub) [inline packages/storebuilder/src/wizards/WizardHeader.tsx:4]: I may be incorrect (and I know the designs show it) but I don't think it makes sense to use the Nexcess logo in the wizard. Please check with Josh, Nell and/or Donaldo to be sure.

**Review by chumpgrub — APPROVED** [review]
I have a question about the use of the Nexcess logo in the wizard, but other than that, this looks great!

**Review by lukefrogger — APPROVED** [review]
LGTM

File: packages/storebuilder/src/wizards/WizardHeader.tsx (hunk 1 above)
Comment (aykanburcak) [reply packages/storebuilder/src/wizards/WizardHeader.tsx:4]: Thanks @chumpgrub I will check that in.

**Pull request merged**